app/scripts/controllers/reviewRegistration.js

```diff
@@ -68,6 +68,182 @@ angular.module('confRegistrationWebApp')
+            $scope.spouseRegistration.registrants[0].firstName = 'Jane';
+            $scope.spouseRegistration.registrants[0].lastName = 'Worner3';
+            $scope.spouseRegistration.registrants[0].answers[1] = {id: uuid(), registrantId: id, blockId: 'b56c9fb7-9643-47d9-9408-ead1bcd6ab31', value: 'janeworner3@hotmail.com'};
+            $scope.spouseRegistration.registrants[0].answers[0] = {id: uuid(), registrantId: id, blockId: '407248f1-6244-49e7-964f-5508634d5ce5', value: {firstName: 'Jane', lastName: 'Worner3'}};
+            ///// Test Data /////
+
+            $scope.spouseExists = true;
+            $rootScope.includeSpouseButtonVisible = true;
+          }
+          else {
+            $scope.spouseRegistration = {};
+            $scope.spouseExists = false;
+            $rootScope.includeSpouseButtonVisible = false;
+            console.log('No current and/or spouse registration found for user with ID.');
+          }
+        }).catch(function(response) {
+          $scope.spouseRegistration = {};
+          $rootScope.includeSpouseButtonVisible = false;
+          console.log('No current and/or spouse registration found for user with ID. Status = ' + response.status);
+      });
+    }
+
+    //Parameters needed to retrieve all current registrants for conference.id
+    var queryParameters = {
+      block: [],
+      //page: 1,
+      //limit: 20,
+      orderBy: 'last_name',
+      order: 'ASC',
+      filter: '',
+      filterPayment: '',
+      filterRegType: '',
+      includeCheckedin: 'yes',
+      includeWithdrawn: 'yes',
+      includeIncomplete: 'yes'
+    };
+
+    // Spouse Registration section visibility
+    $scope.spouseSectionVisible = {
+      spouse: false,
+      current: false,
+      both: false,
+      neither: false
+    };
+
+    //Get all current registrants based on conference.id and queryParameters
+    $http.get('conferences/' + conference.id + '/registrations', {params: queryParameters})
+      .then(function (response) {
+        // (spouseEmailExists = false) then show confirmation page
+        // (spouseEmailExists = true) then do not show confirmation page
+        // var spouseEmailExists;
+        // spouseEmailExists = _.intersection(_.map($scope.spouseRegistration.registrants, 'email'),
+        //     _.map($scope.currentRegistration.registrants, 'email')).length > 0;
+
+        var spouseRegistrantRegistered = false;
+        var currentRegistrantRegistered = false;
+        var spouseRegistrantCompleted = false;
+        var currentRegistrantCompleted = false;
+
+        //Using traditional for loop over angular.forEach because angular.forEach cannot break out of loops
+        //Also didn't think Lodash or underscore would be appropriate here either
+        //Check if spouse already registered
+        for (var i = 0, len = response.data.registrations.length; i < len; i++) {
+          spouseRegistrantCompleted = response.data.registrations[i].completed;
```

Comment:
Absolutely, which is why I put in the break for the other variable I'm setting.  I think to make it more readable I'll place this variable with the other one to eliminate confusion.